<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>601</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>571</width>
    <height>601</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>571</width>
    <height>601</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя и инициалы:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Расчётный счёт отправителя:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FromBillEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Расчётный счёт получателя:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ToBillEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CostEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="InfoLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="BankTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>531</width>
     <height>281</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Имя</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Счёт отправителя</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Счёт получателя</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Сумма</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="SearchComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>245</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Все</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>По счёту отправителя</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>По счёту получателя</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="SearchEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
